driver.sh

```diff
@@ -288,8 +299,8 @@ build_linux() {
     cat ../configs/common.config >> .config
     # Some torture test configs cause issues on x86_64
     [[ $ARCH != "x86_64" ]] && cat ../configs/tt.config >> .config
-    # Disable ftrace on arm32: https://github.com/ClangBuiltLinux/linux/issues/35
-    [[ $ARCH == "arm" ]] && ./scripts/config -d CONFIG_FTRACE
+    # Disable ftrace on arm32 and ppc64be: https://github.com/ClangBuiltLinux/linux/issues/35
+    [[ $ARCH == "arm" || $SUBARCH == "ppc64" ]] && ./scripts/config -d CONFIG_FTRACE
```

Comment:
what's wrong w/ ftrace on ppc64?

Reply:
It was broken at one point...

https://travis-ci.org/shenki/continuous-integration/jobs/474371134

Looks like it works now, I am bisecting the commit that fixes it just for completeness.

https://travis-ci.com/nathanchance/continuous-integration/builds/122211446